python/cugraph/cugraph/tests/traversal/test_bfs.py

```diff
@@ -313,22 +322,27 @@ def get_cu_graph_nx_results_and_params(seed, depth_limit, G, dataset, directed,
 )
 
 
+@pytest.fixture(scope="module")
+def load_traversal_results():
+    return load_resultset("traversal", None)
```

Comment:
What is this returning?  I don't think this fixture needs to return anything.

Reply:
Changed to only call `load_resultset`, rather than returning it, which itself returns nothing.